src/main/java/com/jmb/springfactory/controller/api/LoginController.java

```diff
@@ -0,0 +1,64 @@
+package com.jmb.springfactory.controller.api;
+
+import java.security.Principal;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.security.Principal'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)